src/views/Profile.vue

```diff
@@ -1,13 +1,268 @@
+                  {{ $t('profile.editProfile.save') }}
+                </base-button>
+              </span>
+            </div>
+          </div>
+          <div class="mt-6 px-6 sm:px-10">
+            <div class="divide-y divide-gray-200">
+              <div class="py-4 sm:py-5 sm:grid sm:grid-cols-3 sm:gap-4">
+                <span class="text-sm font-medium text-gray-500">
+                  {{ $t('profile.email.text') }}
+                </span>
+                <span class="flex-grow mt-1 ml-auto flex text-sm text-gray-900 sm:mt-0 sm:col-span-2">
+                  {{ user.email }}
+                </span>
+              </div>
+              <div class="py-4 sm:py-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:pt-5">
+                <span class="text-sm font-medium text-gray-500">
+                  {{ $t('profile.createdAt.text') }}
+                </span>
+                <span class="mt-1 ml-auto flex text-sm text-gray-900 sm:mt-0 sm:col-span-2 flex-grow">
+                  {{ getCreatedAt }}
+                </span>
+              </div>
+              <div class="py-4 sm:py-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:border-b sm:border-gray-200">
+                <span class="text-sm font-medium text-gray-500">
+                  {{ $t('profile.updatedAt.text') }}
+                </span>
+                <span class="flex-grow mt-1 ml-auto flex text-sm text-gray-900 sm:mt-0 sm:col-span-2">
+                  {{ getUpdatedAt }}
+                </span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <transition name="fade">
+      <upload-avatar v-if="showModal" @closeModal="showModal = false" />
+    </transition>
+  </div>
 </template>
 
 <script>
+import {mapActions, mapGetters} from 'vuex';
+import userService from '@/api/userService';
+import get from 'lodash/get';
+import format from 'date-fns/format';
+import formatDistance from 'date-fns/formatDistance'
+import UploadAvatar from '@/components/UploadAvatar';
+
 export default {
-  name: "Profile"
+  name: "Profile",
+
+  components: {
+    UploadAvatar
+  },
+
+  data() {
+    return {
+      edit: false,
+      profile: {
+        first_name: '',
+        last_name: ''
+      },
+      blogs: [],
+      comments: [],
+      showModal: false
+    }
+  },
+
+  computed: {
+    ...mapGetters({
+      user: 'userState'
+    }),
+
+    getCreatedAt() {
+      const date = this.user.created_at.substring(0, 10).split('-');
+
+      return date[0]
+          ? format(new Date(date), 'dd MMM yyyy')
+          : "";
+    },
+
+    getUpdatedAt() {
+      const date = this.user.updated_at.substring(0, 10).split('-');
+
+      return date[0]
+          ? formatDistance(
+              new Date(date),
+              new Date(),
+              { addSuffix: true }
+          )
+          : ""
+    }
+  },
+
+  methods: {
+    ...mapActions(['setUserState', 'setProfileState']),
+
+    get, format, formatDistance,
```

Comment:
If we don't use `format` and `formatDistance` in the template or with `this.format` in the code, we don't have to declare them here